src/utils/mtev_confstr.c

```diff
@@ -157,8 +157,20 @@ mtev_confstr_parse_duration(const char *input, uint64_t *output,
   *output = 0;
   while(true) {
     /* white-space separated, null-terminated */
-    while(*input && isspace(*input))
+    /* sinse space(s) allowed between number and unit string */
+    /* the problem is if a user pass like "10 sec    " */
+    /* we are in infinitive loop according to the new logic line 169 - 170 */
```

Comment:
Not a good idea to put line numbers in comments since when the file gets edited the author will somehow have to remember to come here and edit the line numbers

Reply:
I use line numbers only for this PR which will be removed after